tests/test_git_scraper.py

```diff
@@ -280,7 +281,17 @@ def test_check_for_expired_metrics(expired_repo):
     with open(repositories_file, "w") as f:
         f.write(yaml.dump(repositories_info))
 
-    runner.main(cache_dir, out_dir, None, None, False, True, repositories_file, True, None, None)
+    # Mock `datetime.date.today` so it's a Monday, the only day that
+    # expirations are checked.
+    class MockDate(datetime.date):
+        @classmethod
+        def today(cls):
+            return datetime.date(2019, 10, 14)
+
+    with unittest.mock.patch("probe_scraper.glean_checks.datetime.date", new=MockDate):
```

Comment:
import `unittest.mock`, but use `unittest.mock.patch`? How is that working?

Reply:
This is IIUC a standard way to import a python subpackage.  Could have alternatively done `from unittest import mock` -- happy to change if that's clearer.

Reply:
Oh right, that was my confusion. Carry on!